crates/web-sys/README.md

```diff
@@ -30,19 +30,23 @@ If you don't see a particular web API in `web-sys`, here is how to add it.
    [very bottom](https://w3c.github.io/mediasession/#idl-index) of _that_ page
    is the IDL.
 2. Annotate the functions that can throw with `[Throws]`
-3. Run `cargo run --release --package wasm-bindgen-webidl -- webidls src/features ./Cargo.toml`
-4. Run `git add .` to add all the generated files into git.
-5. Add an entry in CHANGELOG.md like the following
+3. `cd crates/web-sys`
+4. Run `cargo run --release --package wasm-bindgen-webidl -- webidls src/features ./Cargo.toml`
+
+   If formatting fails, you can run `cargo fmt` in the `crates/web-sys` directory. On Windows, you might also want to run `cargo fmt -- --config newline_style=Unix` depending on your git configuration.
```

Comment:
- Considering you are running Windows, could you try and see if prefixing paths with [`\\?\`](https://doc.rust-lang.org/1.83.0/std/path/enum.Prefix.html#variant.Verbatim) fixes the issue on Windows.
- We should set this configuration in the `rustfmt.toml` file (which we don't have, so feel free to create one).